code_samples/querying_with_SQL.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var Client = require('hazelcast-client').Client;
+var Predicates = require('hazelcast-client').Predicates;
+
+function Customer(name, active, age) {
+    this.active = active;
+    this.age = age;
+    this.name = name;
+};
+
+Client.newHazelcastClient().then(function (hazelcastClient) {
+    var client = hazelcastClient;
```

Comment:
no need to define a new variable for the client. Delete this and change `function (hazelcastClient)` to `function (client)`